docs/source/grpo_trainer.md

```diff
@@ -24,6 +24,7 @@ This example demonstrates how to train a model using the GRPO method. We use the
 ></iframe>
 
 Below is the script to train the model. We use PEFT to reduce the memory requirements.
+Note that the actual train batch size is `num_generations * per_device_train_batch_size`; adjusting these values can help prevent OOM errors, as GRPO generates `num_generations` completions for each batch prompt.
```

Comment:
Another option : 
I am wondering if making default batch size to 1 and grad accum to 8 makes sense here

Reply:
Yes agree

Reply:
But can you reformulate here? The actual batch size usually denotes something different (grad accum * per device batch size * num device).
Maybe you can write that "the input tensor for the forward pass has a size of num generation * per device batch size in the batch dim"